spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Services' do
+    it 'can reach out to server with API key and return results' do
+        response = FoodService.ingredient_search("sweet potatoes")
+        expect(response).to be_a Hash
+        expect(response[:foods]).to be_an Array
+    end
```

Comment:
You can also test other keys that you'd expect to get back from this call, for example, you want to make sure that there are keys for the codes, brandowner, description, and ingredients.